### DISCUSSION TITLE
Invoice > payments in

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16664] [How to] [answered]
Created invoice, entered payment in record (exactly the same sum as invoice) but invoice showing "Overpaid"
Where to search for a problem?

**Comment by vovpff** [comment, marked as answer]
I'm not really sure but I think I faced this bug in the past.

Problem what you see 100.00 in invoice but real value stored in db is 99.9999996. In case you add payment in with 100.00 system compare this value with value from db of related invoice. Php compare two values. Condition 100.00 > 99.99999996 means what your invoice overpaid.

Presented values just for reference. You can check in database. Values as 99.99999996 appears when you change quantity of product and apply taxes or discounts

I solved this with rebuilding calculation handler. Then php calculate invoice payment status I round values to 2 digits afted comma.

**Comment by s2b-git** [comment]
Bingo. Invoice showing 3,514.31 but in database is 3514.30800000
I just changed decimals in dbase tables, but this is definitely bug

**Comment by apcloic** [comment]
See also this #13167
